### ISSUE TITLE
Implement Search Button in Main Layout

### ISSUE THREAD

**Issue by brijeshshah13** [brijeshshah13/OpenCI#1]
**Actual Behavior**

Not yet implemented.

**Expected Behavior**

Clicking on the Search button should allow the user to search from the repositories in the current tab, i.e., either `Public` or `Private`.

**Steps to reproduce it**

Click on the search icon in the top right corner of the main layout.

**Comment by hjoshi123** [comment]
@brijeshshah13 is this still open? I would like to work on this.

**Comment by brijeshshah13** [comment]
@hjoshi123 Yes. Please go ahead and implement it. However, do take a note that the Home screen will be changed to display the latest builds instead of the current behavior, i.e., public and private repos, so considering modularity while writing the code.
